snap/snapcraft.yaml

```diff
@@ -156,6 +155,7 @@ parts:
 
   qemu:
     plugin: nil
+    override-pull: ""
```

Comment:
alternatively, create `snap/local` and use that as `source` and remove this stanza. But this implementation is future proof and one line removal away when moving to bases.